Kickstarter-iOS/Views/Controllers/DashboardViewController.swift

```diff
@@ -158,7 +176,7 @@ internal final class DashboardViewController: UITableViewController {
     }
   }
 
-  fileprivate func goToMessages(_ project: Project) {
+  fileprivate func goToMessages(project: Project) {
```

Comment:
@justinswart I am all for using private instead of fileprivate but we have fileprivate class members everywhere in this file. Is is ok if I will switch all of them to private?